documentation/modules/ROOT/pages/integrations/cloudevents.adoc

```diff
@@ -97,6 +101,9 @@ Alternatively, Avro can be used as content type for the `data` attribute:
   "iodebeziumtxId" : "578",
   "iodebeziumlsn" : "33227720",
   "iodebeziumxmin" : null,
+  "iodebeziumtxid": "578",
```

Comment:
Ok, so I'm still not happy about this; but no blocker for now. I've added it to [DBZ-1701](https://issues.redhat.com/browse/DBZ-1701) to get rid of `iodebeziumtxId` if `iodebeziumtxid` is present.